src/main/java/org/ibp/api/rest/labelprinting/SubObservationDatasetLabelPrinting.java

```diff
@@ -285,6 +387,90 @@ public LabelsData getLabelsData(final LabelsGeneratorInput labelsGeneratorInput)
 					}
 
 				} else {
+					final String ObsUnitId = observationUnitRow.getVariables().get(PARENT_OBS_UNIT_ID).getValue();
+					if (STATIC_LOT_FIELD_IDS.contains(field.getId()) || STATIC_TRANSACTION_FIELD_IDS.contains(field.getId())) {
```

Comment:
same here with the OR. Seems to me that it is only adding a new layer of if-elses